server/market/orderrouter.go

```diff
@@ -227,6 +235,10 @@ func (r *OrderRouter) handleLimit(user account.AccountID, msg *msgjson.Message)
 	copy(commit[:], limit.Commit)
 
 	fundingAsset := coins.funding
+	unbookedUnfunded := tunnel.CheckUnfilled(fundingAsset.ID, user)
```

Comment:
Just remembered this is not adequate.  The check needs to be done on all markets...  A loop on the tunnels will be adequate as the CheckUnfilled implementation does nothing if the asset ID is irrelevant to the market.